meteor/client/ui/RundownView.tsx

```diff
@@ -1557,6 +1570,84 @@ export const RundownView = translateWithTracker<IProps, IState, ITrackedProps>((
 					})
 				}
 			})
+			this.autorun(() => {
+				let playlist = RundownPlaylists.findOne(playlistId, {
+					fields: {
+						_id: 1,
+					},
+				})
+				if (playlist) {
+					const rundownIds = playlist.getRundownIDs(undefined, {
```

Comment:
Should this be refactored into a `playlist.getRundownUnorderedIDs()` function?

Reply:
Refactored into a general `getRundownUnorderedIDs()` function.